scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -207,12 +207,17 @@ <h5 class="header">General Institute Settings</h5>
             <td><!-- Gene -->
               <span class="text-body">{{ variant.hgnc_symbols|join(', ') if variant.hgnc_symbols else '--'}}</span>
             </td>
-            <td><!-- Variant -->
-              <a href="{{ url_for('variant.variant',
-                  institute_id=institute._id,
-                  case_name=case.display_name,
-                  variant_id=variant._id) }}">{{ variant.display_name|truncate(15, True) }}
-              </a>
+            <td><!-- Variant link -->
+              {% set var_type_page = {"snv":"variant.variant", "str":"variant.variant", "cancer":"variant.cancer_variant", "sv":"variant.sv_variant", "cancer_sv": "variant.sv_variant"} %}
+              {% for key, page in var_type_page.items() %}
+                {% if variant.category == key %}
+                  <a href="{{ url_for(page,
```

Comment:
Open in other tab perhaps? That is how we do it for other variantS pages.

Reply:
Good idea!

Reply:
Done